jenkins/scripts/files/run_integration_tests.sh

```diff
@@ -123,6 +123,9 @@ fi
 if [ "${TESTS_FOR}" == "feature_tests" ]
 then
   make feature_tests
```

Comment:
what about catching both and running `make ${TESTS_FOR}`

Reply:
done